requirements.txt

```diff
@@ -4,11 +4,12 @@ setuptools
 scipy
 bokeh
 networkx
-toml==0.10.0
+toml
+xlrd==1.2.0
 pyproj
 descartes
 OpenDSSDirect.py==0.3.7
-Shapely
+Shapely==1.6.4
```

Comment:
Does this really need to be tied to 1.6.4? On a Mac running `pip install -e .` on the master branch gave me 1.7.1, which installed with no error. When I switched to this branch and ran the same command, it failed. This package has lots of install issues noted in internet forums. Installing it with conda worked. This behavior will vary widely based on platform.

Reply:
Not sure. I will try and install from scratch and test this out.